cumulusci.yml

```diff
@@ -106,13 +98,15 @@ tasks:
             path: unpackaged/config/dev
             namespace_tokenize: $project_config.project__package__namespace
 
+
     retrieve_config_qa:
         description: Retrieves the current changes in the scratch org into unpackaged/config/qa.
         class_path: cumulusci.tasks.salesforce.sourcetracking.RetrieveChanges
         options:
             path: unpackaged/config/qa
             namespace_tokenize: $project_config.project__package__namespace
 
+
```

Comment:
@kdy618 is there a reason why are adding extra empty lines, for example here?

Reply:
> @kdy618 is there a reason why are adding extra empty lines, for example here?

@xinnanlisf , nope no reason. I will remove it.